synthesizer/src/process/stack/finalize_registers/load.rs

```diff
@@ -76,28 +47,17 @@ impl<N: Network> FinalizeRegisters<N> {
             // If the register is a locator, then return the stack value.
             Register::Locator(..) => stack_value.clone(),
```

Comment:
Can we rename all instances of `stack_value` to `plaintext_value` for readability, now that the type has changed?